app/models/miq_policy.rb

```diff
@@ -369,6 +370,38 @@ def self.seed
     end
   end
 
+  def self.seed_fixtures
+    fixture_file = File.join(FIXTURE_DIR, "miq_policies.yml")
+    fixtures = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    fixtures.each do |policy|
+      rec = seed_attributes(policy.except(:conditions))
+      seed_conditions(rec, policy[:conditions])
+    end
+  end
+
+  def self.seed_attributes(policy)
+    rec = MiqPolicy.find_by_name(policy[:name])
+    if rec.nil?
+      _log.info("Creating MiqPolicy: [#{policy[:name]}] ")
+      rec = MiqPolicy.create(policy)
+    else
+      rec.attributes = policy
```

Comment:
Same as above.  I think it should do the right thing, but maybe we could add a test to ensure the update doesn't clear out existing values that weren't in the hash assigned to attributes.